### PR TITLE
Visualize imu data

### PR DESCRIPTION
**Pull request by aymanhab**
Fixes issue #0

### Brief summary of changes
Provide functionality to visualize IMU data in simbody visualizer

### CHANGELOG.md (choose one)

- no need to update because...
- updated...

The Doxygen for this PR can be viewed at http://myosin.sourceforge.net/?C=N;O=D; click the folder whose name is this PR's number.

---
This change is [Reviewable](https://reviewable.io/reviews/opensim-org/opensim-core/2749)

### CODE REVIEW

File: OpenSim/Simulation/VisualizerUtilities.h [hunk 1]
```diff
@@ -24,9 +24,9 @@
 
 #include "osimSimulationDLL.h"
 #include <OpenSim/Common/TimeSeriesTable.h>
+#include "StatesTrajectory.h"
```
Comment (chrisdembia) [inline OpenSim/Simulation/VisualizerUtilities.h:27]: This could be included in the CPP file instead.

**Comment by chrisdembia** [comment]
This function is not accessible in bindings:

image

**Comment by chrisdembia** [comment]
@aymanhab why are there 2 PRs that include the same changes (showOrientationData())?

**Comment by aymanhab** [comment]
Thanks for pointing this out @chrisdembia The original plan was to get the refactoring of existing functionality (showModel, showMotion, showMocapData) reviewed and merged first, then to add the new functionality being prototyped/refined in a later PR but that doesn't seem likely now so I'll close this PR. Thanks for pointing this out.

**Pull request closed without merging**